app/src/test/java/com/weatherxm/data/datasource/DeviceOTADataSourceTest.kt

```diff
@@ -36,7 +36,7 @@ class DeviceOTADataSourceTest : BehaviorSpec({
     beforeSpec {
         every { firmware.bytes() } returns bytes
         coJustRun { cacheService.setDeviceLastOtaVersion(otaVersionCacheKey, otaVersion) }
-        coJustRun { cacheService.setDeviceLastOtaTimestamp(otaTimestampCacheKey) }
+        coJustRun { cacheService.setDeviceLastOtaTimestamp(otaTimestampCacheKey, any()) }
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/WeatherXM/wxm-android/pull/353#issuecomment-2392359926>#d0cda9</a><span> - 10/03/2024, 09:38 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The use of 'any()' in the mock setup for cacheService.setDeviceLastOtaTimestamp is too permissive and may hide potential issues with the actual value being passed. A similar issue was also found in app/src/test/java/com/weatherxm/data/datasource/UserPreferenceDataSourceTest.kt (line 30-31).
</div>
<div id="fix">
<b>Fix:</b> Replace 'any()' with a more specific matcher that validates the timestamp value. Use 'check' to ensure the passed timestamp is within an acceptable range.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -39 +39 @@
 - coJustRun { cacheService.setDeviceLastOtaTimestamp(otaTimestampCacheKey, any()) }
 + coEvery { cacheService.setDeviceLastOtaTimestamp(otaTimestampCacheKey, check { it in (System.currentTimeMillis() - 1000)..System.currentTimeMillis() }) } just Runs
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged